Automattic/wp-calypso
# BlazePress: Removes BlazePress widget implementation from Stepper

As an alternative approach it was proposed to build the DSP Widget inside Stepper. This is not the way we ended up doing things. Currently the BlazePress widget is a component that lives here: https://github.com/Automattic/wp-calypso/tree/trunk/client/components/blazepress-widget which gets rendered using `BlankCanvas`.

This PR removes the implementation for the DSP from the Stepper.

Serenity: I've added this to our board to track but I've assigned @jjolmo to test this since he knows where the widget is loaded.

### Testing
1. Apply this PR.
2. Verify that the DSP widget loads correctly in all places.
